src/patternfly/demos/Table/table-collapsed-table.hbs

```diff
@@ -1,7 +1,7 @@
 {{#> table table--id="table-collapsed" table--grid="true" table--modifier="pf-m-grid-lg" table--expandable="true" table--attribute='aria-label="Collapsed table example"'}}
```

Comment:
looks like you can remove this file. Looks like it's only used in the "Toolbar expand all (collapsed)" toolbar demo that's removed in this PR.